src/lib.rs

```diff
@@ -793,6 +802,16 @@ impl<T> Sender<T> {
     pub fn same_channel(&self, other: &Sender<T>) -> bool {
         Arc::ptr_eq(&self.shared, &other.shared)
     }
+
+    /// Discards excess capacity in the internal queue.
+    pub fn shrink_to_fit(&self) {
+        self.shared.shrink_to_fit();
+    }
```

Comment:
I think a less precise name might be better here to account for implementation changes: there's no saying whether we'll want to continue to use `std`'s queues in the future.